packages/ComponentLibrary/src/components/BasicModal/index.tsx

```diff
@@ -31,6 +37,25 @@ const Modal: React.FC<ModalIProps> = ({
     setOpen(false);
   };
 
+  const getGradientStyles = () => {
```

Comment:
I think this would be better like:

`const gradientStyles = !backgroundGradient ? {} : { '&::before': { ... } }`